site/content/integrate/admin-guide/admin-personal-access-token/_index.md

```diff
@@ -35,25 +35,25 @@ Then select **Save**.
 
 ## Revoking a Personal Access Token
 
-A personal access token can be revoked by deleting the token from either the user's account settings or from the System Console. Once deleted, all sessions using the token are deleted, and any attempts to use the token to interact with the Mattermost server are blocked.
+A personal access token can be revoked by deleting the token from either the user's profile or from the System Console. Once deleted, all sessions using the token are deleted, and any attempts to use the token to interact with the Mattermost server are blocked.
```

Comment:
Shouldn't this be "profile settings" and not just "profile"? I feel like "user's profile" has a different meaning than the profile settings

Reply:
Great catch! Thank you!